modules/services/postgresql/default.nix

```diff
@@ -5,6 +5,27 @@ with lib;
 let
   cfg = config.services.postgresql;
 
+  postgresqlAndPlugins = pg:
+    if cfg.extraPlugins == [] then pg
+    else pkgs.buildEnv {
+      name = "postgresql-and-plugins-${(builtins.parseDrvName pg.name).version}";
+      paths = [ pg pg.lib ] ++ cfg.extraPlugins;
+      # We include /bin to ensure the $out/bin directory is created which is
+      # needed because we'll be removing files from that directory in postBuild
+      # below.
+      pathsToLink = [ "/" "/bin" ];
+      buildInputs = [ pkgs.makeWrapper ];
+      postBuild =
+        ''
+          mkdir -p $out/bin
```

Comment:
`$out/bin` is assumed to already exists per your comment above. It this necessary?

Reply:
it wasn't necessary, fixed. Thanks!